libr/bin/format/elf/elf.h

```diff
@@ -39,6 +39,7 @@ typedef struct r_bin_elf_symbol_t {
 	int last;
 	bool in_shdr;
 	bool is_sht_null;
+	bool is_value;
```

Comment:
The `is_value` name mirrors [ELF symbol table documentation](https://docs.oracle.com/cd/E23824_01/html/819-0690/chapter6-79797.html), specifically the documentation for the `st_value` field.

> Maybe even adding a comment like `when true, the paddr value is not present and vaddr is set to...` or something.

Another issue here is whether `RBinSymbol`'s vaddr (not to be confused with `RBinElfSymbol` above) can be overloaded like this, or whether `RBinSymbol` should also get an extra `is_value` field. The current way is because of memory concerns.

Reply:
I see. But `st_value` is an Elf_Addr, while this is a bool and it's not clear what it does (in my opinion at least).

Reply:
ok comment added in 63c593a663147a0ea96b5da424fc8957430010ad